packages/web/src/pages/modals/Modals.tsx

```diff
@@ -129,16 +129,20 @@ const Modals = () => {
 
   return (
     <>
-      {commonModals.map(([modalName, Modal]) => {
-        return <AppModal key={modalName} name={modalName} modal={Modal} />
-      })}
       <PasswordResetModal />
       <FirstUploadModal />
       <UnloadDialog />
       <CollectibleDetailsModal />
-
-      {!isMobile && (
+      {isMobile ? (
+        <>
+          <ConnectedMobileOverflowModal />
+          <UnfollowConfirmationModal />
+        </>
+      ) : (
         <>
+          {commonModals.map(([modalName, Modal]) => {
```

Comment:
just double-checking this logic - we don't need these modals on mobile-web?

Reply:
I would hope not since none of them are formatted to work correctly, will double check git history though

Reply:
yeah we chillin